.github/workflows/npm-run-package.yml

```diff
@@ -0,0 +1,46 @@
+name: 'npm run package'
+# Main use case: repackage when Dependabot updates a dependency
+on:
+  push:
+    branches:
+      - 'dependabot/**'
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Process this branch'
+        required: false
+        type: string
+
+jobs:
+  npm-run-package-and-push: # make sure build/ci work properly
+    runs-on: ubuntu-latest
+    if: github.event_name == 'workflow_dispatch' || github.event.repository.owner.login == 'git-for-windows'
+    environment: git-for-windows-ci-push
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.repository.full_name }}
+          ref: ${{ inputs.branch }}${{ github.event.ref }}
+          token: ${{ secrets.GIT_FOR_WINDOWS_CI_PUSH }}
+      - run: npm ci
+      - run: npm run build
+      - run: npm run lint
+      - run: npm run format && git diff-files
```

Comment:
There is no `format` scriot in `package.json`

Reply:
Done. There's also no `npm run package`.

Reply:
The commot message still talks about `npm run package`

Reply:
> The commot message still talks about `npm run package`

Oops.